ydb/src/client_builder.rs

```diff
@@ -25,6 +25,7 @@ static PARAM_HANDLERS: Lazy<Mutex<HashMap<String, ParamHandler>>> = Lazy::new(||
         m.insert("token".to_string(), token);
         m.insert("token_cmd".to_string(), token_cmd);
         m.insert("token_metadata".to_string(), token_metadata);
+        m.insert("token_password".to_string(), password);
```

Comment:
better to replace it for "token_static_password"
where static - is approach for receive token and password is param for static type